### DISCUSSION TITLE
Add explicit warnings to every build version

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#557] [General]
Before the Disk Utility or install. app is officially solved and can handle dual boot, mark a text on each download that builds it "⚠️Note, this version is not recommended for dual boot, it is very likely that your information thus lost”

**Comment by borisnezlobin** [comment]
something like that would be very helpful (maybe modified a bit to show that the disk will be erased completely, dual boot won't work?)

Maybe I could work on this?

**Reply by louies0623** [reply]
You can add it here to let users know. Of course, I would like to have a "⚠️" icon to make it easy to identify.
https://github.com/helloSystem/Utilities/blob/042e74f75a9040fbca6650691964a5f054669d26/Utilities/Install%20helloSystem.app/Resources/install.py#L595

**Reply by borisnezlobin** [reply]
Ok. I'll do this sometime tomorrow. I'm not actually using helloSystem at the moment, so I might not be able to test the changes.

**Reply by borisnezlobin** [reply]
I finally got to this, and I'm almost done (it was pretty easy). I do have one question though - will the "⚠️" icon display properly + consistently with the helloSystem font?

edit: also, is there a specific color I should use for the warning? a red/yellow color? is there a helloSystem palette?

**Reply by louies0623** [reply]
The icon may need to use the 64x64icon of elementary os (to be converted to png). Emoji cannot be used. The Nimbus Sans defaulted by the system may be missing words. As for the text, I don’t know the standard color, but I would recommend using bold.

**Reply by louies0623** [reply]
I'm sorry that the position I gave is incorrect. This is the style I want to present.
![images - 2023-10-08T162658 470](https://github.com/helloSystem/hello/assets/44593430/c9caa14c-a659-4943-b9ef-00765323c500)

**Reply by borisnezlobin** [reply]
Ok, I'll do this
